spec/controllers/api/companies_controller_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'rails_helper'
+
+RSpec.describe Api::CompaniesController, type: :controller do
+  let!(:user) { create(:user, :admin) }
+
+  describe 'action #index' do
+    context 'with valid tokens' do
+      it 'should respond with companies data' do
+        request.headers['Authorization'] = "Token token=#{user.auth_token}"
```

Comment:
you have the same line in every spec, maybe move it to before_action or to some helper method.